src/components/layout/Header/HeaderIconsSvg.tsx

```
@@ -0,0 +1,144 @@
+export const SearchIcon = () => (
+  <svg
+    className="h-5 w-5 text-gray-400"
+    fill="none"
+    stroke="currentColor"
+    viewBox="0 0 24 24"
+  >
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)